components/file_attachment/file_attachment.tsx

```diff
@@ -27,7 +25,9 @@ import {
 import FilenameOverlay from './filename_overlay';
 import FileThumbnail from './file_thumbnail';
 
-interface Props {
+import {PropsFromRedux} from './index';
+
+interface Props extends PropsFromRedux{
```

Comment:
Some nitpicking about styling
```suggestion
interface Props extends PropsFromRedux {
```

Reply:
✅  Updated as per request/discussion